setup.py

```diff
@@ -66,11 +66,13 @@
 XPALIB_DIR = "cextern/xpa-2.1.15"
 CONF_H_NAME = os.path.join(XPALIB_DIR, "conf.h")
 
+# Output from Cythonizing
+CYTHON_SOURCE = "wrappers/xpa.c"
+
 # check if cython is available.
 try:
     from Cython.Build import cythonize
     cythonize("wrappers/xpa.pyx")
-    CYTHON_SOURCE = "wrappers/xpa.c"
     have_cython=True
 except ImportError:
     print("Unable to load Cython")
```

Comment:
sure, if you have time to clean it up that would be great. I haven't followed helpers closely the last year. Note the cython is needed for anyone using ds9. There's no way to tell at the start, but make sure the check is there for anyone trying to open an imexam object (which uses ds9 by default) that if cython isn't present it returns a reasonable error telling them to use the ginga window or install cython.